src/main/java/org/broadinstitute/hellbender/tools/picard/vcf/GatherVcfs.java

```diff
@@ -91,8 +91,10 @@ protected Object doWork() {
     /** Checks (via filename checking) that all files appear to be block compressed files. */
     private boolean areAllBlockCompressed(final List<File> input) {
         for (final File f : input) {
-            // TODO move/expose these methods elsewhere, since VariantContextWriterFactory is deprecated
-            if (VariantContextWriterFactory.isBCFOutput(f) || !VariantContextWriterFactory.isCompressedVcf(f)) return false;
+            if (f != null) {
```

Comment:
Lets write a test for this since we obviously have some questions.  

To do so you'll have to elevate areAllBlockCompressed to default access level.  Then annotate it with `@VisibleForTesting` so we know why it has a weird permissions level.

Make a new test file called GatherVcfsUnitTest, and make a test with the following cases:

+ null input
+ List of null
+ List of "test.bcf"
+ List of "test.vcf"
+ List of "test.vcf.gz"
+ List of "test1.vcf","test2.vcf.gz"

Run the original code and use it to decide what the output values are.  Then try your condition and my condition and see if either of them pass the test.